src/css/components/_content.tabs.scss

```diff
@@ -37,11 +37,12 @@
     
     @include font-size(16px, 1);
     @include for-size("desktop-up"){
-      display: inline-block;  
+      display: table-cell;  
       width: px-to-rem(125px);
       height: px-to-rem(50px);
       word-break: break-word;
       text-align: center;
+      vertical-align: middle;
```

Comment:
This will likely conflict with the changes I pushed up for this. It's fine either way we just need to determine which method we want to use.